### ISSUE TITLE
In Windows, if focus switches before launch with -f, window stays on top even if -T is not used.

### ISSUE THREAD

**Issue by jef-pearlman** [Genymobile/scrcpy#552]
In Windows 10, if you launch scrcpy with -f (full screen), and switch windows (e.g., with alt-tab) before the scrcpy window opens, then the scrcpy window stays on top even if you haven't used the -T (stay on top) option.  

Focus does not transfer to the new scrcpy window, meaning you can still type in whatever window you were using, but it will be hidden behind the scrcpy window.  You can also use alt-tab to switch focus to other windows and use them, but of course they're hidden as well.

This behavior occurs with both the console and non-console versions, and whether you launch from a command line, the Run window, or a Windows shortcut.  It also occurs if you switch back to the original launching window before the scrcpy window opens.  It does not occur if -f is not used.

Using ctrl-f twice to cancel and re-set full screen mode fixes the issue (and can function as a workaround).

**Comment by rom1v** [comment]
Thank you for the report.

The fullscreen flag is just delegated to SDL:

https://github.com/Genymobile/scrcpy/blob/a41dd6c79f542925c5db7f1ee51d27b65ba832ec/app/src/screen.c#L294

SDL probably itself just delegates to the window manager of Windows, so I guess there is nothing we can do at _scrcpy_ level.

**Comment by jef-pearlman** [comment]
Just noting one aspect of this behavior that I misinterpreted before: if you alt-tab to the scrcpy window and back out, it actually does start letting other windows on top of scrcpy.  This is less obvious than you might think because the full-screen on-top window starts as the _last_ one in the alt-tab stack so I wasn't actually alt-tabbing to it before.

(I thought this had changed in v1.9, but I went back and I don't think it has.)

I don't think it changes anything about your response; I just wanted to clarify in here in case it's useful later.
